### PR TITLE
Feature/can create playlist from app

### PR DESCRIPTION
**Pull request by TharickABDUL**
> **IMPORTANT !!!!**
> 
>  **Each PR has to be deployed before merging in order to pass all checks**
> 
> To deploy the PR, follow these steps:
>
>>  If you want to deploy `dwaves-api` on `test` environemnt, add the label `deploy-dwaves-api-test` to the PR. 
>
>>  If you want to deploy `dwaves-app` or `dwaves-website` on `test` environemnt, add the label `deploy-dwaves-api-prod` to the PR.

---

## What & How
* Can Create a Playlist
* Display all playlists of an user
* Can Like/Dislike music

## Tickets
* [Can GET Playlist](https://app.clickup.com/t/860q9xn8y)
* [Can POST Playlist](https://app.clickup.com/t/3640tff)
* [Likes music](https://app.clickup.com/t/3640x3p)

### CODE REVIEW

File: frontend/dwaves-app/src/App.tsx [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 import { Loader, ExploPlayer, Sidebar, Alert, Footer } from 'components'
-import { Player, Explorer, Album, Download, ModalLogin, Profile } from 'views'
+import {Player, Explorer, Album, Download, ModalLogin, Profile, PlaylistPage, Playlist} from 'views'
 
 import { useEffect, useRef, useState } from 'react'
 import { BrowserRouter as Router, Routes, Route } from 'react-router-dom'
 import { Icon } from "components/shared";
 import { responseRequest, Music, AlbumDetail } from 'models'
 import { PlayPause, PlayRandomSong } from 'songs/listenMusic'
+import axios from "axios";
```
Comment (Wbebey) [inline frontend/dwaves-app/src/App.tsx:10]: remove this semi column

File: frontend/dwaves-app/src/components/ContentAlbum.tsx [hunk 2]
```diff
@@ -1,18 +1,21 @@
 import 'styles/ContentAlbum.scss'
 import { Icon } from 'components/shared'
 import { Link, useParams } from 'react-router-dom'
-import { useEffect, useState } from 'react'
+import React, { useEffect, useState } from 'react'
```
Comment (Wbebey) [inline frontend/dwaves-app/src/components/ContentAlbum.tsx:4]: do you really need to import react here ?

File: frontend/dwaves-app/src/components/ContentOfPlaylist.tsx
Comment (Wbebey) [inline frontend/dwaves-app/src/components/ContentOfPlaylist.tsx:1]: change the file name to `PlaylistContents`

File: frontend/dwaves-app/src/components/PlaylistsOfUser.tsx
Comment (Wbebey) [inline frontend/dwaves-app/src/components/PlaylistsOfUser.tsx:1]: same for the component name, change to `UserPlaylist` or `PlaylistUser`

File: frontend/dwaves-app/src/components/PopularSongOfArtist.tsx
Comment (Wbebey) [inline frontend/dwaves-app/src/components/PopularSongOfArtist.tsx:1]: same here ofr the component name

File: frontend/dwaves-app/src/components/ContentAlbum.tsx (hunk 2 above)
Comment (TharickABDUL) [reply frontend/dwaves-app/src/components/ContentAlbum.tsx:4]: bah oui, logique !

**Review by alexandre-pinon — APPROVED** [review]
C'est vraiment parce qu'on a plus le temps...

**Review by alexandre-pinon — APPROVED** [review]
Faut rebase avant de demander une reveiw en fait...

**Pull request merged**